### PR TITLE
Fix (#6) https://github.com/NukkitX/SynapsePM/issues/6

### PR DESCRIPTION
**Pull request by KCodeYT**

### CODE REVIEW

**Comment by KCodeYT** [comment]
Tested: true

**Comment by KCodeYT** [comment]
Fixing the issue: true

**Comment by KCodeYT** [comment]
Adding the ability of getting the channel: true

**Comment by KCodeYT** [comment]
Am I good: true

File: src/synapsepm/Synapse.php [hunk 1]
```diff
@@ -295,7 +296,11 @@ public function handleDataPacket(DataPacket $pk) {
                 break;
             case SynapseInfo::PLUGIN_MESSAGE_PACKET:
                 /** @var PluginMessagePacket $pk */
-                $this->server->getPluginManager()->callEvent(new SynapsePluginMessageReceiveEvent($this, $pk->channel, $pk->data));
+                $event = new SynapsePluginMessageReceiveEvent($this, $pk->channel, $pk->data);
+                try {
+                    $event->call();
+                } catch(ReflectionException $e) {
```
Comment (Creeperface01) [inline src/synapsepm/Synapse.php:302]: I think you shouldn't ignore it

**Pull request closed without merging**